release-notes/3.0/preview/api-diff/preview7/standalone-packages/3.0-preview7-standalone-packages_System.md

```diff
@@ -0,0 +1,176 @@
+# System
```

Comment:
We don't have a "System" standalone package, so this file name is a bit confusing.